packages/jest-core/src/__tests__/SearchSource.test.ts

```diff
@@ -531,4 +531,65 @@ describe('SearchSource', () => {
       }
     });
   });
+
+  describe('findRelatedSourcesFromTestsInChangedFiles', () => {
+    const rootDir = path.resolve(
+      __dirname,
+      '../../../jest-runtime/src/__tests__/test_root',
+    );
+
+    beforeEach(done => {
+      const {options: config} = normalize(
+        {
+          haste: {
+            hasteImplModulePath: path.resolve(
+              __dirname,
+              '../../../jest-haste-map/src/__tests__/haste_impl.js',
+            ),
+            providesModuleNodeModules: [],
+          },
+          name: 'SearchSource-findRelatedSourcesFromTestsInChangedFiles-tests',
+          rootDir,
+        },
+        {} as Config.Argv,
+      );
+      Runtime.createContext(config, {maxWorkers, watchman: false}).then(
+        context => {
+          searchSource = new SearchSource(context);
+          done();
+        },
+      );
+    });
```

Comment:
```suggestion
    beforeEach(async () => {
      const {options: config} = normalize(
        {
          haste: {
            hasteImplModulePath: path.resolve(
              __dirname,
              '../../../jest-haste-map/src/__tests__/haste_impl.js',
            ),
            providesModuleNodeModules: [],
          },
          name: 'SearchSource-findRelatedSourcesFromTestsInChangedFiles-tests',
          rootDir,
        },
        {} as Config.Argv,
      );
      const context = await Runtime.createContext(config, {maxWorkers, watchman: false})

      searchSource = new SearchSource(context);
    });
```

diff looks big, but async await instead of `.then` and `done`